src/stores/AccountStore.ts

```diff
@@ -3,6 +3,21 @@ import type { Consent, Customer, CustomerConsent } from '#types/account';
 import type { Offer } from '#types/checkout';
 import type { PaymentDetail, Subscription, Transaction } from '#types/subscription';
 
+type Features = {
```

Comment:
These are not account features, they are account service features. I'd suggest we leave them as readonly consts on the service or controller instead of putting them into a store. They should never change throughout the life of the app unless the config is reloaded to trigger new modules.

Reply:
Moved to controller's props.